point to Discuss for new architecture (#7463)
circleci/circleci-docs: jekyll/_cci2/building-docker-images.md

## Overview
{: #overview }

> The underlying implementation for Remote Docker is in the process of being updated. **There is no action required from users.** All further content on this page refers to the implementation of Remote Docker that is in the process of being phased out. Once all jobs have been migrated to the new implementation, the content currently on this page will become outdated and will be replaced with information based on the new architecture.   
> <br>
> Visit the [Discuss post](https://discuss.circleci.com/t/setup-remote-docker-architecture-change/45303) to learn more details regarding the new architecture and to follow updates regarding the rollout. 

Reviewer: ```suggestion
The underlying implementation for remote Docker is in the process of being updated. **There is no action required.** All content on this page refers to the existing implementation of remote Docker that is being phased out. When all jobs have been migrated to the new implementation, the content on this page will be updated to reflect the new architecture.   
<br>
Visit the [Discuss post](https://discuss.circleci.com/t/setup-remote-docker-architecture-change/45303) to learn more about the new architecture, and to follow updates regarding the rollout. 
```
